lib/src/judo_floating_card.dart

```diff
@@ -0,0 +1,210 @@
+  final Function closeAction;
+  final double height = 144;
+  final double width = 444;
+
+  @override
+  Widget build(BuildContext context) {
+    if(messageStack.isNotEmpty) {
+      final ThemeData theme = Theme.of(context);
+
+      return Positioned.fill(
+        bottom: 15,
+        child: Align(
+          alignment: Alignment.bottomCenter,
+          child: Stack(
+            clipBehavior: Clip.none,
+            alignment: Alignment.center,
+            children: [
+              ..._floatingCardsStack(messageStack),
+              Container(
+                width: width,
+                height: height,
+                decoration: BoxDecoration(
+                  boxShadow: [BoxShadow(
+                    color: Colors.black26,
+                    blurRadius: 60,
+                  )],
+                  borderRadius: BorderRadius.vertical(
+                      bottom: Radius.circular(20)
+                  ),
+                  color: _getBackgroundColor(messageStack.first.messageLevel),
+                ),
+              ),
+              Positioned(
+                  right: 350,
+                  child: _getIcon(messageStack.first.messageLevel, theme),
+              ),
+              _getText(messageStack.first.messageLevel),
+              Positioned(
+                left: 132,
+                bottom: 24,
+                child: ConstrainedBox(
+                  constraints: BoxConstraints(
+                    maxWidth: 280,
+                  ),
+                  child: Container(
+                    height: 24,
+                    child: Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 20),
+                      child: Text(
+                        messageStack.first.actionCallName ?? '',
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                          color: _getTextColor(messageStack.first.messageLevel, theme),
+                        ),
+                        overflow: TextOverflow.fade,
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              Positioned(
+                  top: 2,
+                  right: 2,
+                  child: Material(
+                    type: MaterialType.transparency, // InkSplash occurs on the closest ancestor Material widget.
+                    child: IconButton(
+                      onPressed: closeAction,
+                      icon: Icon(
+                        Icons.close,
+                        size: 18,
+                      ),
+                      splashRadius: 20,
+                    ),
+                  )
+              )
+            ],
+          ),
+        ),
+      );
+    }
+    return SizedBox.shrink();
+
+  }
+
+  List<Widget> _floatingCardsStack(final List<FloatingMessage> messageStack) {
+    if (messageStack.length < 2) return [SizedBox.shrink()];
```

Comment:
Please add a comment above this line about the reason why we are returning the above!

Reply:
SizedBox.shrink() means nothing in flutter world.